reference/system-databases/cluster-log.md

```diff
@@ -0,0 +1,73 @@
+---
+title: CLUSTER_LOG
+summary: Learn the `CLUSTER_LOG` cluster log table.
+category: reference
+---
+
+# CLUSTER_LOG
+
+You can query cluster logs on the `CLUSTER_LOG` cluster log table. By pushing down query conditions to each node, the impact of the query on cluster performance is less than that of the `grep` command.
+
+To get the logs of the TiDB cluster before v4.0, you need to log in to each node to summarize logs. This cluster log table in 4.0 provides the global and time-ordered log search result, which makes it easier to track full-link events. For example, by searching logs according to the `region id`, you can query all logs in the life cycle of this Region; similarly, by searching the full link log through the slow log's `txn id`, you can query the flow and the number of keys scanned by this transaction at each node.
+
+{{< copyable "sql" >}}
+
+```sql
+desc cluster_log;
+```
+
+```
++----------+---------------------------+------+------+---------+-------+
+| Field    | Type                      | Null | Key  | Default | Extra |
++----------+---------------------------+------+------+---------+-------+
+| TIME     | varchar(32)               | YES  |      | NULL    |       |
+| TYPE     | varchar(64)               | YES  |      | NULL    |       |
+| INSTANCE | varchar(64)               | YES  |      | NULL    |       |
+| LEVEL    | varchar(8)                | YES  |      | NULL    |       |
+| MESSAGE  | var_string(1024) unsigned | YES  |      | NULL    |       |
++----------+---------------------------+------+------+---------+-------+
+5 rows in set (0.00 sec)
+```
+
+Field description:
+
+* `TIME`: The time to print the log.
+* `TYPE`: The node type. The optional values are `tidb`, `pd`, and `tikv`.
+* `INSTANCE`: The service address of the node.
+* `LEVEL`: The log level.
+* `MESSAGE`: The log content.
+
+> **Note:**
+>
+> + All fields of the cluster log table are pushed down to the corresponding node for execution. So to reduce the overhead of using the cluster log table, specify as many conditions as possible. For example, the `select * from cluter_log where instance='tikv-1'` statement only executes the log search on the `tikv-1` node.
```

Comment:
ditto. @reafans PTAL It seems that it should be `instance`. @TomShawn Please pay attention to all `node`s in this PR after @reafans confirms.

Reply:
OK

Reply:
1. The format of instance is `ip:port`, which represent a TiDB or TiKV instance.
2. The node is not accurate, maybe host is better, it represents a physical node, which contains some instance, and its format like `ip`.
3. All the node in this pr need be changed to instance.

Reply:
All `node` is replaced in the recently added system table documents.